tools/gcp_init/installer.py

```diff
@@ -553,15 +562,28 @@ def RemoteOsSetup(host=None):
         fab.sudo('apt-get -y update; apt-get -fy dist-upgrade')
 
         log.info('installing remote dependencies')
-        fab.sudo('apt-get -fy install python-virtualenv git liblzma-dev '
-                 'rsync python-pip python-wheel socat pv vim-nox sudo '
-                 'wget curl python-openssl')
+        fab.sudo('apt-get -fy install git liblzma-dev sudo '
+                 'cpio pax rsync socat pv vim-nox wget curl '
+                 'tmux screen tcpdump '
+                 'python-pip python-wheel python-virtualenv '
+                 'python-openssl ')
 
         log.info('upgrading remote pip')
         fab.sudo('pip install --upgrade pip')
         log.info('installing remote wheel')
         fab.sudo('pip install wheel')
 
+        log.info('adding turbinia user')
+        fab.sudo('grep -q turbinia /etc/group || '
+                 'groupadd -g {0:d} turbinia'.format(tcfg.TURBINIA_GID))
```

Comment:
I think both user and group names should added to the config instead of being hardcoded here, so the systemd init script will know which user/group Turbinia will run as.

Reply:
I withdraw my previous comment because systemd does not support dynamic user and group in the service template. By default, the Turbinia packaging distribution should ship with turbinia/turbinia as the user/group. So, users who wish to run Turbinia as a different user/group can manually edit the script to meet their needs.